src/components/theme/FavBoard/FavBoardView.jsx

```diff
@@ -333,9 +334,22 @@ const FavBoardView = (props) => {
         <Portal node={document.getElementById('toolbar')}>
           <Toolbar
             inner={
-              <Link className="item" to="/boards">
-                <Icon name={backSVG} size="30px" className="contents circled" />
-              </Link>
+              <>
+                <Link className="item" to="/boards">
+                  <Icon
+                    name={backSVG}
+                    size="30px"
+                    className="contents circled"
+                  />
+                </Link>
+                <Link className="item" exact to="/data-maps-and-tools/metadata">
```

Comment:
Not sure about the icon, it could be confusing (with the website search). Add a title on hover something like "Back to resource catalogue"